### ISSUE TITLE
Cleanup redundant content and navigation on storefront and ux platform

### ISSUE THREAD

**Issue by marcelbertram** [porsche-design-system/porsche-design-system#1751]
Scope
---
With ux.porsche.com we created a space for static content (like Accessibility, Design Onboarding, Design Worfklows). This content should be removed from the Storefront and the navigation updated.

Notes  
---
- Update static pages in storefront, e.g. Support, Help
- Make navigation hierarchy simpler
- Ensure that URL Bookmarks still work if links are removed

Acceptance criteria
---
- [ ] Remove redundant information and improve storefront navigation

Subtasks
---
- [ ] Task
